src/views/statutory/Stats.vue

```diff
@@ -293,13 +293,25 @@ export default {
       }
     },
     byQuorumData () {
-      const present = this.stats.by_body.length * 100 / this.bodies.length
+      const localsMap = {}
+      for (const body of this.bodies) {
+        if (['antenna', 'contact antenna', 'partner'].includes(body.type)) {
+          localsMap[body.id] = true
+        }
+      }
+
+      const total = Object.keys(localsMap).length
+      const present = this.stats.by_body
+        .filter((body) => body.type in localsMap)
```

Comment:
will that work? I'm not sure if it will, but if so, that's okay

Reply:
Yes it works on my local device

Reply:
When I apply as AEGEE-Eindhoven, it gets added to the quorum, but when I do so as ITC, it does not show up.

Reply:
Awesome